modules/client/src/connext.ts

```diff
@@ -227,6 +230,30 @@ export const connect = async (opts: ClientOptions): Promise<IConnextClient> => {
   log.debug("Registering subscriptions");
   await client.registerSubscriptions();
 
+  // TODO: should we have a flag for this?
```

Comment:
as in a flag as to whether this app is already installed or an event thrown or what

Reply:
A flag to enable async deposits to a token.